Tests/AMPStorageTests.m

```diff
@@ -23,17 +31,15 @@ - (void)setUp {
 
 - (void)tearDown {
     [super tearDown];
-    NSString* dir = [AMPStorage getAppStorageAmpDir:@""];
-    NSFileManager *fileMgr = [NSFileManager defaultManager];
-    NSArray *fileArray = [fileMgr contentsOfDirectoryAtPath:dir error:nil];
-    for (NSString *filename in fileArray)  {
-        [fileMgr removeItemAtPath:[dir stringByAppendingPathComponent:filename] error:NULL];
-    }
+    [Amplitude cleanUp];
 }
 
 - (void)testHasFileStorage {
```

Comment:
The previous way shouldn't work because after fixed the tearDown, all the created instance directories will be removed, there shouldn't have a pass for @"INSTANCE_NAME" storage before storing any event. If it's a clean env, the first-time run fails, and the following try will all passed because we didn't remove the @"INSTANCE_NAME" file storage in the previous tearDown. @dantetam

Reply:
I agree with this. The `@"INSTANCE_NAME"` or the directory above it (the xctest directory I talked about in my most recent iOS PR) should be deleted